.mailmap

```diff
@@ -404,6 +406,8 @@ Tim Smart <timehandgod@gmail.com> <tim@fostle.com>
 Tim Smart <timehandgod@gmail.com> <timehandgod@gmail.com>
 Timothy Leverett <zzzzBov@gmail.com> Timothy <zzzzBov@gmail.com>
 Tobias Nießen <tniessen@tnie.de> <tniessen@users.noreply.github.com>
+Tom Atkinson <atkinson.tommy@nsoft.com> <atkinson.tommy@nhome.ba>
+Tom Atkinson <atkinson.tommy@nsoft.com> <atkinson.tommy@nsoft.ba>
```

Comment:
@Nibbler999 Which address do you prefer to have listed in `AUTHORS`?

Reply:
atkinson.tommy@nsoft.com is my preferred address